The grand reopening Part 2: Add overview page (#432)
trixi-framework/TrixiParticles.jl: docs/src/index.md

  - VTK

## Examples
```@raw html
<table align="center" border="0">
  <tr>
    <td align="center">
      <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/683e9363-5705-49cc-9a5c-3b47d73ea4b8" style="width: 80% !important;"/><br><figcaption>2D Dam Break</figcaption>
    </td>
    <td align="center">

>       <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/c10faddf-0400-47c9-b225-f5d286a8ecb8" style="width: 80% !important;"/><br><figcaption>Moving Wall</figcaption>

Author: I have not created a picture of the 3D version.

>       <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/c10faddf-0400-47c9-b225-f5d286a8ecb8" style="width: 80% !important;"/><br><figcaption>Moving Wall</figcaption>

Reviewer: I was thinking about the 2D version, but I guess it would be great to have a 3D simulation on the landing page.
Then leave it like this for now, and I'll add a 3D picture that I have lying around.

    </td>
  </tr>
  <tr>
    <td align="center">
      <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/e05ace63-e330-441a-a391-eda3d2764074" style="width: 80% !important;"/><br><figcaption>Oscillating Beam</figcaption>
    </td>
    <td align="center">
      <img src="https://github.com/trixi-framework/TrixiParticles.jl/assets/10238714/ada0d554-e0ba-44ed-923d-2b77ef252258" style="width: 80% !important;"/><br><figcaption>Dam Break with Elastic Plate</figcaption>
    </td>
  </tr>
</table>
```

## Quickstart
1. [Installation](@ref installation)
2. [Getting started](@ref getting_started)

> If you have **any** questions concerning **TrixiParticles.jl** you can also reach us on [this public slack channel](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g) also feel free to open an issue with your question.

Author: 1. ok
2. Yes but stuff is lost on slack. Also i am not sure how many people like slack... Any way I don't see us drowning in either slack msg or issues with questions right now.

> If you have **any** questions concerning **TrixiParticles.jl** you can also reach us on [this public slack channel](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g) also feel free to open an issue with your question.

Reviewer: 2. Then I'm okay with issues as well. But it seems that most of the Julia community is focused on Slack.
```suggestion
If you have any questions concerning TrixiParticles.jl you can join our community [on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g) or open an issue with your question.
```
